### PR TITLE
add command to set the KRaft mode in the operator

### PR DESCRIPTION
**Pull request by ngtszman**
### Type of change

_Select the type of your PR_

- Documentation

### Description

make more clear how to enable KRaft mode (zookeeper-less) 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [Y] Write tests
- [Y] Make sure all tests pass
- [Y] Update documentation

### CODE REVIEW

File: examples/kafka/nodepools/README.md [hunk 1]
```diff
@@ -22,7 +22,12 @@ The [`kafka-with-kraft.yaml`](./kafka-with-kraft.yaml) deploys a Kafka cluster w
 The [`kafka-with-dual-role-kraft-nodes.yaml`](./kafka-with-dual-role-kraft-nodes.yaml) deploys a Kafka cluster with one pool of KRaft nodes that share the _broker_ and _controller_ roles.
 
 _NOTE: To use this example, you have to enable the `UseKRaft` feature gate in addition to the `KafkaNodePools` feature gate._
-
+```
+kubectl set env deployments/strimzi-cluster-operator STRIMZI_FEATURE_GATES=+UseKRaft,+KafkaNodePools -n kafka
```
Comment (fvaleri) [inline examples/kafka/nodepools/README.md:26]:
```suggestion
kubectl set env deployments/strimzi-cluster-operator STRIMZI_FEATURE_GATES=+UseKRaft,+KafkaNodePools -n <NAMESPACE>
```

File: examples/kafka/nodepools/README.md [hunk 2]
```diff
@@ -22,7 +22,12 @@ The [`kafka-with-kraft.yaml`](./kafka-with-kraft.yaml) deploys a Kafka cluster w
...
+```
```
Comment (fvaleri) [inline examples/kafka/nodepools/README.md:27]: There is no need to create two code blocks.
```suggestion
```

**Review by fvaleri — COMMENTED** [review]
Thanks for the PR. Left some comments.

I wonder if we should also include the `UnidirectionalTopicOperator` feature gate, adding the topic operator to all KRaft examples in this folder.

**Review by scholzj — COMMENTED** [review]
Thanks for the PR. However, this is a bit oversimplified:
* The operator might be installed using OPeratorHUb or Helm in which case the configuration is completely different.
* The operator might run in a different namespace in which case this will not work.
* And some other situations like this

So this is intentionally left open because you need to figure out what is the right way to do it for you.

There are also several examples and not just `kafka-with-kraft.yaml` so that does not always apply either.

**Review by scholzj — COMMENTED** [review]
Discussed on the Community call of 19.10.2023: We do not really want to have the specific instructions in the README file. Can you please replace it with a link to the relevant documentation section talking about feature gates and how to enable them?

**Comment by scholzj** [comment]
Discussed on the community call on 30.11.2023: There were no changes t this since October. We should clsoe this and it can be reopened again if needed.

**Pull request closed without merging**